### PR TITLE
Fix formatting

### PR DESCRIPTION
**Pull request by ryancoughlin**
- ABC
- Replace underscores with dashes

### CODE REVIEW

**Comment by ryancoughlin** [comment]
@mordecai174 @samccone Fixed in https://github.com/ryancoughlin/mo/commit/50c783b7d62ab487acb1545501f5efa1b24d097f

**Comment by ryancoughlin** [comment]
@samccone Also see this: https://github.com/ryancoughlin/mo/commit/50c783b7d62ab487acb1545501f5efa1b24d097f#diff-2afa16dbc8588799e9290e01f19b6425R85 - formatting for the `{}` - yay? nay?

**Comment by ryancoughlin** [comment]
@samccone Squashed.

**Comment by samccone** [comment]
@ryancoughlin doc updates and this will be good to go

**Comment by ryancoughlin** [comment]
Updated docs: https://github.com/mojotech/mo/commit/08fcfa9da684e8116d5a1c8f7715a8f4d8546229

**Comment by samccone** [comment]
sweet one commit and :ship: and I will cut a new release

**Comment by ryancoughlin** [comment]
Pushed.

**Comment by samccone** [comment]
@ryancoughlin mind rebasing off of master I made it so the tests will now run again

**Comment by markalfred** [comment]
hey guys

**Comment by markalfred** [comment]
:dog2:

**Comment by samccone** [comment]
ping @ryancoughlin

**Comment by ryancoughlin** [comment]
How do I rebase from a fork? Never done this before.

**Comment by ryancoughlin** [comment]
`origin` isnt the same right?

**Comment by ryancoughlin** [comment]
And its technically not a remote

**Comment by ryancoughlin** [comment]
Or I dont have it listed as one.

**Comment by samccone** [comment]
oh yeah ok do this

`git add real git@github.com:mojotech/mo.git`
`git fetch real`
`git rebase real/master`
`git push -f`

File: _mo.scss [hunk 1]
```diff
@@ -111,15 +109,15 @@
 }
 
 @mixin ellipsis($width: false) {
-  @if $width { width: $width };
   overflow:       hidden;
-  white-space:    nowrap;
   text-overflow:  ellipsis;
+  white-space:    nowrap;
```
Comment (samccone) [inline _mo.scss]: @ryancoughlin looks like the order stuff you changed here causes a few tests to fail should be a snap to fixup let me know if you are stuckkk this is 95% of the way there

**Comment by samccone** [comment]
Thanks @ryancoughlin 
manually pulled in to the 0.4.0 release :white_check_mark: 

https://github.com/mojotech/mo/commit/aac8fa491607d6b7069d07292aedd01b728ef238

**Comment by samccone** [comment]
@ryancoughlin Added houndCI :dog: 

:+1: :+1: :+1:

**Comment by ryancoughlin** [comment]
Thanks for this, was caught up yesterday. So this is all good?

—

**Comment by samccone** [comment]
all good now thanks for the work

**Comment by ryancoughlin** [comment]
@samccone Awesome! Saves so much time for little things!

**Comment by ryancoughlin** [comment]
No sweat. What's next?

**Comment by ryancoughlin** [comment]
:8ball:

**Pull request closed without merging**